src/cellRenderers/SimpleCell.js

```diff
@@ -320,6 +320,9 @@ function renderSingleLineText(gc, config, val, leftPadding, rightPadding, hideRi
         x += halignOffset;
         y += Math.floor(config.bounds.height / 2);
 
+        const fontHeight = gc.getTextHeight(config.font).height;
+        y += fontHeight * 0.075;
```

Comment:
This seems pretty arbitrary - I thought we were going to center the text in the row?

Reply:
We are centering the text already, but there is a bit of space under the text. This change is to eliminate the space.